server/sonar-web/src/main/js/apps/coding-rules/rule/rule-issues-view.js

```diff
@@ -49,22 +49,23 @@ export default Marionette.ItemView.extend({
         const projectBase = r.components.find(component => component.uuid === project.val);
         return {
           ...project,
-          name: projectBase != null ? projectBase.longName : ''
+          name: projectBase != null ? projectBase.longName : '',
+          issuesUrl: getComponentIssuesUrl(projectBase.key, {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') TypeError can be thrown as "projectBase" might be null or undefined here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS2259)